entrypoint.sh

```diff
@@ -49,4 +54,6 @@ elif [ "$APPLICATION_TYPE" = "compliance-import-remediations" ]; then
 	exec bundle exec rake import_remediations --trace
 elif [ "$APPLICATION_TYPE" = "compliance-import-ssg" ]; then
 	exec bundle exec rake ssg:import_rhel_supported --trace
+else
+    echo "Application type '$APPLICATION_TYPE' not supported"
```

Comment:
Nice :laughing:

Reply:
it helped with having something when deploying this locally